cylc/uiserver/tests/conftest.py

```diff
@@ -20,8 +20,10 @@
 import logging
 from pathlib import Path
 from shutil import rmtree
+import shutil
```

Comment:
`rmtree` is already imported:

```suggestion
from shutil import rmtree
```